anchor-reference-server/src/main/java/org/stellar/anchor/reference/AnchorReferenceServer.java

```diff
@@ -24,15 +28,19 @@ public class AnchorReferenceServer implements WebMvcConfigurer {
 
   static ConfigurableApplicationContext ctx;
 
-  public static ConfigurableApplicationContext start(int port, String contextPath) {
+  public static ConfigurableApplicationContext start(
+      Map<String, String> envMap, int port, String contextPath) {
+    if (envMap == null) {
+      envMap = new HashMap<>();
+    }
+    envMap.put("server.port", String.valueOf(port));
```

Comment:
These are nice refactors. 

💡 Just a kind reminder. These classes will be deprecated and replaced with the Kotlin reference server.

Reply:
Yep, we will need to build this functionality into the Kotlin server once we merge the two.